docs/_documentations/troubleshooting.md

```diff
@@ -119,9 +119,30 @@ When attempting to install Codewind in IntelliJ, you cannot locate the Codewind
 
 This error occurs because the custom plug-in repository link contains an additional space.
 
-**Workaround**
+**Workaround:**
 Remove the extra space from the custom plug-in repository link.
 
+<!--
+Action/Topic: Installing Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/2563
+0.11.0: New
+-->
+## Without a keychain or keyring, local Codewind crashes when certain commands are used
+Local Codewind needs to use a keychain, also called a keyring, when certain commands are used.
```

Comment:
It may be useful here to mention the actual error message. On linux we have seen: 
`dbus: invalid bus address (invalid or unsupported transport`
and
`The name org.freedesktop.secrets was not provided by any .service files`

This way if the user searches this page for their error message they can find it.

Also, Remote codewind uses the keychain too.

Reply:
Maybe
` ## Without a keychain or keyring, Codewind fails to run certain commands`